.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: "Updatecli: Dependency Management"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 */2 * * *'
+  repository_dispatch:
+    types: [kubewarden-controller-release]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
we should use git hashs to pin the GHA dependencies, instead of just the @v3 tag. Example: https://github.com/kubewarden/helm-charts/blob/main/.github/workflows/update-dependencies.yaml#L13-L17

Reply:
ok, thanks. How do I discover that hash? I can't see how to get it. Is it from somewhere here? 

https://github.com/actions/checkout/releases/tag/v3.0.0

Reply:
The original says checkout@v3. Would that then be the earliest i.e. 3.0.0 or the last available 3.x.x?

and similarly for updatecli-action@v2

Reply:
yes, that's correct. Not a merge blocker at all, we pin the GHA dependencies for the other repos to be sure on consuming GHA dependencies that aren't tampered with. For this repo is not as important now that I think about it.

Reply:
Resolving as done.